.circleci/config.yml

```diff
@@ -23,27 +23,32 @@ commands:
       - run:
           name: Generate and apply migration
           command: npx prisma migrate dev
-  setup_docker_compose:
+  setup_docker_compose_dependencies:
     steps:
       - setup_remote_docker:
           version: 20.10.14
       - run:
           name: Run images of services declared in docker-compose.yml
-          command: docker-compose up -d
-  create_test_env:
+          command: docker-compose --profile dependencies up -d
+  create_env:
     steps:
       - run:
-          name: Create test.env file
+          name: Create .env file
```

Comment:
If you have to create 2 environment files then modify names here accordingly, because right now you are not creating just `.env` file only.